lib/msal-browser/src/interaction_handler/RedirectHandler.ts

```diff
@@ -41,6 +32,20 @@ export class RedirectHandler extends InteractionHandler {
         return window;
     }
 
+    navigateToRequestUrl(): void {
+        if (window.parent === window) {
+            const loginRequestUrl = this.browserStorage.getItem(TemporaryCacheKeys.ORIGIN_URI);
```

Comment:
I think this should be passed in as a parameter. That way this functions becomes "pure"/idempotent, which makes it easier to test and understand.

Reply:
done, please review the updates.